lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -612,6 +612,9 @@ def add_auth_service(xml, payment_method, options)
       def add_normalized_threeds_2_data(xml, payment_method, options)
         threeds_2_options = options[:three_d_secure]
         cc_brand = card_brand(payment_method).to_sym
+
+        return if !threeds_2_options[:cavv].present? && infer_commerce_indicator?(options, cc_brand)
```

Comment:
Perhaps use `empty?` instead of `!present?` to increase legibility?

```suggestion
        return if threeds_2_options[:cavv].empty? && infer_commerce_indicator?(options, cc_brand)
```